azure/template.json

```diff
@@ -52,11 +52,12 @@
             "defaultValue": "[utcNow('yyMMddHHmmss')]"
         }
     },
-    "variables": {
-        "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
-        "resourceNamePrefix": "[toLower(concat('das-', parameters('resourceEnvironmentName'),'-', parameters('serviceName')))]",
-        "appServiceName": "[concat(variables('resourceNamePrefix'), '-as')]"
-    },
+  "variables": {
+    "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
+    "resourceNamePrefix": "[toLower(concat('das-', parameters('resourceEnvironmentName'),'-', parameters('serviceName')))]",
+    "appServiceName": "[concat(variables('resourceNamePrefix'), '-as')]",
+    "adminServiceConfigNames": "SFA.DAS.AdminService"
```

Comment:
This is odd - see my comment below about the DfeSignIn config row key - which I'm sure the system is currently using - but if it was not here already how - was it using it?

Reply:
Resolved, and included.